solidity/dashboard/src/connectors/utils.js

```diff
@@ -14,11 +14,18 @@ export const getEthereumTxObj = (txData, chainId) => {
   return new EthereumTx(txData, { common })
 }
 
-// EIP-155 https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
-// v = CHAIN_ID * 2 + 35 => CHAIN_ID = (v - 35) / 2
+/**
+ * This function returns the chain id from `v` param based on to the EIP-155[1]
+ * [1]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+ * Formula: v = CHAIN_ID * 2 + 35 => CHAIN_ID = (v - 35) / 2
+ *
+ * @param {Uint8Array} vInHex v param in hex.
```

Comment:
I would personally add hyphen between the name of the parameter and its description. As stated in [JSDoc](https://jsdoc.app/tags-param.html#names-types-and-descriptions) the hyphen is not necessary but it makes it more readable in my opinion. I'll leave it up to you to decide.

Reply:
I'm actually puzzled, is this a uint8array or a string (which is what "in hex" implies to me)?

On the subject of the dash, I think the dash becomes less clarifying if the parameter description is a full sentence (because it typically starts with a capital letter). Eg here, "The signature `v` value to extract the current chain from." or similar.